pontoon/machinery/views.py

```diff
@@ -113,6 +117,30 @@ def machine_translation(request):
         return HttpResponseBadRequest('Bad Request: {error}'.format(error=e))
 
 
+def machine_translation_caighdean(request):
+    """Get translation from Caighdean machine translation service."""
+    try:
+        entityid = int(request.GET['id'])
+    except (MultiValueDictKeyError, ValueError) as e:
+        return HttpResponseBadRequest('Bad Request: {error}'.format(error=e))
```

Comment:
This is simply a question: when do you use a Form, and when do you not? I guess if there's just one field, then you don't bother creating a new class for handling that? Do you have a rule for when to use a Form?

Reply:
imho - use a form by default - unless you are using pretty much the unchanged resolved keywords from the url

Reply:
i think this would be better implemented in a form, incidentally